shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -362,6 +368,7 @@ - (UIAccessibilityTraits)accessibilityTraits {
     traits |= UIAccessibilityTraitAdjustable;
   }
   if ([self node].HasFlag(blink::SemanticsFlags::kIsSelected) ||
+      [self node].HasFlag(blink::SemanticsFlags::kIsToggled) ||
```

Comment:
Maybe add a TODO that this doesn't give us the on/off announcements that we really want?

Reply:
Done